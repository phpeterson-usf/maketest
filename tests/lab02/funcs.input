$(TESTS_DIR)/funcs.ntc
