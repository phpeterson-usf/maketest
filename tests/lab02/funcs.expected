TK_IDENT("print")
TK_LPAREN("(")
TK_STRLIT("Hello, ntcalc!")
TK_RPAREN(")")
TK_EOL("")
TK_IDENT("println")
TK_LPAREN("(")
TK_RPAREN(")")
TK_EOL("")
TK_EOT("")
